### DISCUSSION TITLE
Is it possible to toggle remote branches on the git graph?

### DISCUSSION THREAD

**Discussion by GP4cK** [jesseduffield/lazygit#4021] [Q&A] [answered]
On a project with a lot of branches, I'd rather have only the local branches show for clarity...

**Comment by mark2185** [comment, marked as answer]
You can use a custom `git log` command to show all of the branches in the main window:
```yaml
git:
  allBranchesLogCmd: git log --graph --all --color=always --abbrev-commit --decorate --date=relative  --pretty=medium --whatever-else-you-want
```

[Link to the documentation](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md).
